sailthru_content/services/tests/test_sailthru_api_service.py

```diff
@@ -0,0 +1,133 @@
+#! /usr/bin/env python
+import json
+import mock
+import unittest
+
+from factory import fuzzy
```

Comment:
@clintonb Sorry, you are right. My comment was directed to @rlucioni. But you are correct by pointing out `fuzzy` is getting deprecated. I'll make the change to use `faker` instead